Dockerfile

```
@@ -1,20 +1,20 @@
 # See:
 # https://github.com/devcontainers/images/tree/main/src/javascript-node
-ARG NODE_VERSION=18
+ARG NODE_VERSION=20
 FROM mcr.microsoft.com/devcontainers/javascript-node:${NODE_VERSION}
 
 WORKDIR /usr/src/app
 ADD . /usr/src/app
```

Hadolint

[error] Use COPY instead of ADD for files and folders

(DL3020)